platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/impl/SystemPropertiesAdmin.java

```diff
@@ -234,7 +257,20 @@ public void writeSystemProperties(Map<String, String> updatedSystemProperties) {
                 LOGGER.warn("Unable to write system user attribute file for hostname update.", e);
             }
         }
+    }
 
+    /**
+     * Overwrite attributes with those from GuestClaimsHandlerExt so system high is set.
+     */
+    private void addGuestClaimsProfileAttributes(Map<String, Object> json) {
+        Map<String, Object> selectedProfileAttributes =
+                guestClaimsHandlerExt.getSelectedClaimsProfileAttributes();
```

Comment:
Is this setting system high to exactly what guest is getting set to, or can they be different (they need to be able to be different).